client/transactions/list/index.tsx

```diff
@@ -670,16 +684,24 @@ export const TransactionsList = (
 					method: 'POST',
 				} );
 
-				createNotice(
-					'success',
-					sprintf(
-						__(
-							'Your export will be emailed to %s',
-							'woocommerce-payments'
-						),
-						userEmail
-					)
-				);
+				if ( response.export_id ) {
+					setTimeout( async () => {
```

Comment:
We could use an interval to continue to poll the endpoint to see if the file is ready, e.g. check every 1s to a maximum of 5s.

Reply:
Confused about how much wait time is ideal.

Reply:
Me too. I'm thinking we check every ~1s, and set a max retries param.

I've explored this in #10212

Reply:
Merged back into this PR & resolving